src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const { id, email } = user;
+  const token = jwt.sign({ id, email }, process.env.JWT_KEY);
```

Comment:
Ensure that `process.env.JWT_KEY` is set in your environment variables. This key is crucial for signing and verifying access tokens.